contracts/interfaces/IBatcher.sol

```diff
@@ -12,6 +12,10 @@ interface IBatcher {
     error BatcherNotImplementedError();
     error BatcherBalanceMismatchError(UFixed18 oldBalance, UFixed18 newBalance);
 
+    function reserve() external view returns (address);
```

Comment:
`function RESERVE() external view returns (IEmptySetReserve)` should suffice here, so we can leverage Solidity's automatic getters for public variables

Reply:
Same for the below 2

Reply:
Ah ok, I misinterpreted the idea as us wanting to have a utility to extract the addresses from USDC/DSU.